unittests/linuxliketests.py

```diff
@@ -1863,3 +1863,8 @@ def test_complex_link_cases(self):
         self.assertIn('build t9-e1: c_LINKER t9-e1.p/main.c.o | libt9-s1.a libt9-s2.a libt9-s3.a\n', content)
         self.assertIn('build t12-e1: c_LINKER t12-e1.p/main.c.o | libt12-s1.a libt12-s2.a libt12-s3.a\n', content)
         self.assertIn('build t13-e1: c_LINKER t13-e1.p/main.c.o | libt12-s1.a libt13-s3.a\n', content)
+
+    def test_sp_options(self):
```

Comment:
I wonder if this makes more sense in unittests/platformagnostictests.py :)

Reply:
This only makes sense when pkg-config is installed.  Pure Visual Studio and pure Xcode do not have it.

Reply:
We are testing top-level options, not pkgconfig per se. Also, the pkgconfig *module* doesn't care whether pkg-config programs are installed or not :) it is purely about writing out the text file format.

Reply:
Renamed. I'm not going to spend time moving this around. It is sufficient that it is run in _some_ environment. Let's not bikeshed this to death but instead get the release out.